docs/ops/broadcast_rules.md

```diff
@@ -0,0 +1,50 @@
+# Broadcast Rules For Elementwise Operations {#openvino_docs_ops_broadcast_rules}
+
+The purpose of this document is to provide a set of common rules which are applicable for ops using broadcasting.
+
+## Description
+
+Broadcast allows to perform element-wise operation for inputs with different shapes. Smaller input is *stretched* into an array with the same shape as the larger input. As a result both inputs have compatible shapes.
+
+## Rules
+
+1. Right aligned dimensions of the two tensors are compared elementwise.
+2. Smaller tensor is prepended with dimension(s) of size 1 in order to have the same shape as the larger tensor.
+3. After alignment two tensors are compatible when:
+   * both are equal or
+   * one of the dimensions is 1.
+4. Tensor with dimension of size 1 will be implicitly broadcasted to match the size of the second tensor.
+
```

Comment:
This is a description of the "numpy" broadcasting mode. But we also have "bidirectional" (in the *Broadcast* operation specification). Please, describe it here as well.

Reply:
Bidirectional broadcast is added. @lazarevevgeny please verify description.